### ISSUE TITLE
Item#getName() always returns "Unknown"

### ISSUE THREAD

**Issue by L3W3S** [PocketDreams/SequinLand#29]
### Issue description

Whenever I create an item and fetch its name (e.g. TNT) through Item#getName(), it always returns "Unknown".

### Steps to reproduce the issue

`Item item = new Item(276, 0, 1);
Server.getInstance().broadcastMessage(item.getName());`

### OS and versions
Nukkit Version: 1.0-dev
Java Version: 1.8.0_111
Host OS: Windows
Client Edition: Windows 10
Client Version: 1.0.3

**Comment by MrPowerGamerBR** [comment]
@L3W3S can you try reproducing this issue in Nukkit?

**Comment by L3W3S** [comment]
@MrPowerGamerBR Yes, it also happens in Nukkit. I reported it here because I'd most likely get Trump's tax returns before a fix at Nukkit repo.

**Comment by MrPowerGamerBR** [comment]
@L3W3S and another question, this happens with EVERY block except items, right?

**Comment by L3W3S** [comment]
@MrPowerGamerBR Items and blocks. I've tried TNT, stone, dirt, diamond sword, diamond hoe etc.

**Comment by MrPowerGamerBR** [comment]
@L3W3S oh okay, that's strange.

**Comment by MrPowerGamerBR** [comment]
@L3W3S like, there isn't no reason for that to happen, for blocks? Maybe, but for items?

Unless if the item (example: Diamond Sword) constructor is ignored, then yeah, the name will be "Unknown".

I may try debugging and fixing this issue later, I'm busy with [another project](https://github.com/PocketDreams/SequinLand-Glowstone) so... yeah.

But again, very strange, and I hope it isn't difficult to be fixed.

**Comment by L3W3S** [comment]
@MrPowerGamerBR I had a look at the source and I saw that a direct constructor to Item does not fetch the name from the item (e.g. ItemArrow) depending on the ID because ItemArrow constructs Item with the name parameter "Arrow".. if that makes any sense.

**Comment by DaMatrix** [comment]
@L3W3S I'm not able to reproduce, I've got this code:
https://github.com/2pocket2edition/Nukkit2p2e/blob/master/src/main/java/cn/nukkit/Player.java#L3811
and it works fine.

**Comment by DaMatrix** [comment]
getCustomName calls getName if there's no name for the item

**Comment by L3W3S** [comment]
@DaMatrix This only occurs when creating an Item, not fetching the item in their hand because this is usually ItemDiamondSword and not Item.

**Comment by MrPowerGamerBR** [comment]
*facepalm*

@L3W3S yeah, that will cause the issue, I didn't see that you were creating the item using the "Item" constructor.

Wouldn't it be better if you used the "correct" item class then? (Example: ```new ItemDiamond(...)```), this would fix your issue.

Fixing the issue is a bit difficult since then I would need to check the item ID in the "getName()" method.

**Comment by L3W3S** [comment]
@MrPowerGamerBR Tiny issue is that I am creating the items via a command (/i <id>).

**Comment by MrPowerGamerBR** [comment]
@L3W3S yeah... that makes things more difficult.

I need to check where Nukkit does that "Item" -> "ItemDiamond" change.
